core/io/file_access_pack.cpp

```diff
@@ -47,29 +53,75 @@ Error PackedData::add_pack(const String &p_path, bool p_replace_files, uint64_t
 	return ERR_FILE_UNRECOGNIZED;
 }
 
+Error PackedData::remove_pack(const String &p_path) {
+	if (!is_pack_loaded(p_path)) {
+		return ERR_FILE_UNRECOGNIZED;
+	}
+
+	Vector<PathMD5> reload_packs;
+	for (HashMap<PathMD5, PackedFile, PathMD5>::Iterator E = files.begin(); E; ++E) {
+		PackedFile pf = E->value;
+		if (pf.pack.name != p_path) {
+			continue;
+		}
+
+		PathMD5 pmd5 = pf.pack.replaced_pack;
+		if (pmd5.set) {
+			if (!reload_packs.has(pmd5)) {
+				reload_packs.push_back(pmd5);
+			}
+		}
+
+		remove_path(pf.filepath);
+		files.remove(E);
+	}
+	remove_loaded_pack(p_path);
+
+	// For reloading files unloaded by more recent packs, we simply need to reload packs with replace_files disabled
```

Comment:
```suggestion
	// For reloading files unloaded by more recent packs, we simply need to reload packs with replace_files disabled.
```
Code style nitpick, same in places below

Reply:
Fixed, thanks!